lib/kubernetes-deploy/kubernetes_resource/bugsnag.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Bugsnag < KubernetesResource
+    TIMEOUT = 1.minute
+
+    def initialize(name, namespace, context, file)
+      @name = name
+      @namespace = namespace
+      @context = context
+      @file = file
+      @secret_found = false
+    end
+
+    def sync
+      _, _err, st = run_kubectl("get", type, @name)
+      @found = st.success?
+      if @found
+        secrets, _err, st = run_kubectl("get", "secrets", "--output=jsonpath='{.items[*].metadata.name}'")
+        @secret_found = secrets.split.any?{|s| s.end_with?("-bugsnag")}
```

Comment:
yes, but that's unless they call the resource something else, so I'd say it's really `<%= @name %>-bugsnag`, but that's going to change soon too. I also dislike duplicating the same assumption in several places (source, templates, deploys)